sklearn/preprocessing/data.py

```diff
@@ -184,6 +184,7 @@ def scale(X, axis=0, with_mean=True, with_std=True, copy=True):
 
 
 class MinMaxScaler(BaseEstimator, TransformerMixin):
+
```

Comment:
Oh, my bad. I installed a pep8 formatter for my sublime but it seems to have formatted more than I thought.